tests/rules/test_key_ordering.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Adrien Vergé
```

Comment:
# Copyright (C) 2017 Johannes F. Knauf
;-)

Reply:
Updated the feature branch